grype/db/v6/installation/curator_test.go

```diff
@@ -220,6 +226,10 @@ func TestCurator_Update(t *testing.T) {
 	t.Run("error during activation: cannot move dir", func(t *testing.T) {
 		c := setupCuratorForUpdate(t, withWorkingUpdateIntegrations())
 		mc := c.client.(*mockClient)
+		// nop hydrator
+		c.hydrator = func(string) error {
```

Comment:
hydrator could be optional, checked for nil where it's used instead of requiring empty functions like this

Reply:
I can change it to nil check